var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -29,6 +29,9 @@ class Esmf(MakefilePackage):
     # Develop is a special name for spack and is always considered the newest version
     version("develop", branch="develop")
     # generate chksum with 'spack checksum esmf@x.y.z'
+    version("8.7.0b04", commit="609c81179572747407779492c43776e34495d267")
+    version("8.6.1", sha256="dc270dcba1c0b317f5c9c6a32ab334cb79468dda283d1e395d98ed2a22866364")
+    version("8.6.1b04", commit="64d3aacc36f2d4d39255eb521c34123903cc0551")
```

Comment:
We don't need to add that either, since 8.6.1 is there

Reply:
removed 8.6.1b04